docs/source/basic-usage.rst

```diff
@@ -58,25 +58,27 @@ into higher level representations of WebSocket events. In ``<APPLICATION
 GLUE>``, you need to write code to process these events. The
 :class:`WSConnection <wsproto.connection.WSConnection>` class contains a
 generator method :meth:`events <wsproto.connection.WSConnection.events>` that
-yields WebSocket events. To send a message, you call the :meth:`send_data
-<wsproto.connection.WSConnection.send_data>` method.
+yields WebSocket events. To send a message, you call the :meth:`send
+<wsproto.connection.WSConnection.send>` method.
 
 Connecting to a WebSocket server
 --------------------------------
 
-Begin by instantiating a connection object. The ``host`` and ``resource``
-arguments are required to instantiate a client. If the WebSocket server is
-located at ``http://myhost.com/foo``, then you would instantiate the connection
-as follows::
+Begin by instantiating a connection object in the client mode and then
+create a :class:`Request <wsproto.events.Request>` class to send. The
+Request must specify ``host`` and ``resource`` arguments. If the
```

Comment:
resource or target? because below in the example you used `target` (previously it was `resource`)